src/lib/storage/table.cpp

```diff
@@ -267,6 +268,40 @@ void Table::set_table_statistics(const std::shared_ptr<TableStatistics>& table_s
 
 std::vector<IndexStatistics> Table::indexes_statistics() const { return _indexes; }
 
+const std::vector<TableConstraintDefinition>& Table::get_unique_constraints() const { return _constraint_definitions; }
+
+void Table::add_unique_constraint(const std::vector<ColumnID>& column_ids, bool primary) {
```

Comment:
const bool? I don't know how we handle const's that probably don't matter.

Reply:
Absolutely.